generators/run/templates/run

```diff
@@ -84,6 +84,7 @@ pip_cache_volume="${CANONICAL_WEBTEAM_PIP_CACHE_VOLUME:-canonical-webteam-pip-ca
 <% if (db) { %>db_container="${project}-db"
 db_volume="${project}-db"
 network_name="${project}-net"<% } } %>
+run_serve_docker_opts="${RUN_SERVE_DOCKER_OPTS:-}"
```

Comment:
Where we use other environment variables, we namespace them `CANONICAL_WEBTEAM`, could we do the same?